ui/client/common/JsonUtils.js

```diff
@@ -5,9 +5,7 @@ import Flattenizer from "flattenizer";
 // They should be removed as soon as friendly UI components for complex input objects are ready
 
 export function tryStringify(input) {
-  return _.isString(input) ? input :
-    _.isObject(input) ? JSON.stringify(input) :
-      input;
+  return _.isString(input) ? input : _.isObject(input) ? JSON.stringify(input) : input;
```

Comment:
if we change this, maybe let's add some parentheses... I'm still not sure if I can parse it correctly :)

Reply:
this is only text formatter.. :)